### PR TITLE
Comment out volunteers

### PR DESCRIPTION
**Pull request by pscott**
Commented out instead of greying. Dunno what we prefer?

### CODE REVIEW

File: src/pages/Tickets.tsx [hunk 1]
```diff
@@ -3,8 +3,8 @@ import React, { useState } from 'react';
 import { Container, Flex, Text, Heading, UnlockModal, Link } from '@components';
 import {
   KYBER_PAYWALL,
-  TICKET_FIAT_PAYWALL,
-  VOLONTEER_APPLICATION,
+  // TICKET_FIAT_PAYWALL,
+  // VOLUNTEER_APPLICATION,
```
Comment (pscott) [inline src/pages/Tickets.tsx:7]: Commented out those because they weren't used anymore

**Comment by GuillaumeRx** [comment]
I would prefer greying it ous as we did for the others :)

File: src/pages/Tickets.tsx (hunk 1 above)
Comment (GuillaumeRx) [reply src/pages/Tickets.tsx:7]: You can delete them if they are not used

**Review by GuillaumeRx — CHANGES_REQUESTED** [review]
Requested changes in the other comment :)

**Comment by pscott** [comment]
@GuillaumeRx For some reason I missed out the notification two days ago.. my bad. We need to merge this one ASAP (once it's good on your end), as people keep on submitting volunteer applications heheh

File: src/pages/Tickets.tsx [hunk 2]
```diff
@@ -132,7 +127,7 @@ const Tickets = () => {
             >
               Sold out
             </Text>
-            <Text
+            {/* <Text
```
Comment (GuillaumeRx) [inline src/pages/Tickets.tsx:130]: Please use the same method as the other disabled buttons for this one

**Review by GuillaumeRx — CHANGES_REQUESTED** [review]

**Review by GuillaumeRx — APPROVED** [review]
After discussion we're keeping the `NEED TO BUY DAI` commented; lgtm

**Pull request merged**